src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,97 @@
-     */
+    
     public LocalTime addHours(LocalTime localTime, Integer hoursToAdd) {
-        return LocalTime.now();
+        return localTime.plusHours(hoursToAdd);
     }
-
-    /**
-     * Given the time and the number of minutes to add, return the changed time.
-     */
+    
     public LocalTime addMinutes(LocalTime localTime, Integer minutesToAdd) {
-        return LocalTime.now();
+        return localTime.plusMinutes(minutesToAdd);
     }
-
-    /**
-     * Given the time and the number of seconds to add, return the changed time.
-     */
+    
     public LocalTime addSeconds(LocalTime localTime, Integer secondsToAdd) {
-        return LocalTime.now();
+        return localTime.plusSeconds(secondsToAdd);
     }
-
-    /**
-     * Given the date and the number of weeks to add, return the changed date.
-     */
+    
     public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
-        return LocalDate.now();
+        return localDate.plusWeeks(numberOfWeeks);
     }
-
-    /**
-     * Given a random `someDate` date, return one of the following Strings:
-     *     - "`someDate` is after `currentDate`"
-     *                  if `someDate` is in the future relating to the `current date`;
-     *     - "`someDate` is before `currentDate`"
-     *                  if `someDate` is in the past relating to the `current date`;
-     *     - "`someDate` is today"
-     *                  if `someDate` is today;
-     */
+    
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        if (someDate.isAfter(LocalDate.now())) {
+            return someDate.toString() + " is after " + LocalDate.now().toString();
+        }
+        if (someDate.isBefore(LocalDate.now())) {
+            return someDate.toString() + " is before " + LocalDate.now().toString();
+        }
+        return someDate.toString() + " is today";
     }
-
-    /**
-     * Given a String representation of a date and some timezone,
-     * return LocalDateTime in this timezone.
-     */
+    
     public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone) {
-        return LocalDateTime.now();
+        return ZonedDateTime.parse(dateInString)
+                .withZoneSameInstant(ZoneId.of(zone)).toLocalDateTime();
     }
-
-    /**
-     * Given some LocalDateTime, return an OffsetDateTime with the local time offset applied
-     * (`+02:00` for Ukraine).
-     *
-     * Example: we receive a LocalDateTime with a value `2019-09-06T13:17`.
-     *          We should return the OffsetDateTime with a value `2019-09-06T13:17+02:00`,
-     *          where `+02:00` is the offset for our local timezone.
-     *
-     * OffsetDateTime is recommended to use for storing date values in a database.
-     */
+    
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return localTime.atOffset(ZonedDateTime.now().getOffset());
```

Comment:
Travis may failed because of this, try to use 
```suggestion
        ZoneOffset zoneOffset = ZoneOffset.of(OFFSET_UA);
        return OffsetDateTime.of(localTime, zoneOffset);
```
where OFFSET_UA is a string "+02:00"

Reply:
Hmm... quite new for me. Thanks for tip